src/App.js

```diff
@@ -4,23 +4,30 @@ import EditNoteModal from "./Components/UI/Modal/EditNoteModal";
 import LeftNavBar from "./Components/UI/Navigation/LeftNavBar";
 import NavBar from "./Components/UI/Navigation/NavBar";
 import BodyWrapper from "./Components/UI/Wrapper/BodyWrapper";
-import { usePebbleNote } from "./Context/PebbleNoteProvider";
+import { useModal, usePebbleNote } from "./Context";
 import LandingPage from "./Pages/LandingPage/LandingPage";
 import HomePage from "./Pages/HomePage/HomePage";
```

Comment:
Suggestion: an index file for pages could be created and all pages can be imported from there.

Reply:
The index file is there, it might have taken the path automatically. Will fixed this.